tuw_nlp/grammar/lexicon.py

```diff
@@ -275,20 +420,38 @@ def handle_obl_case(self, parent_dep, parent_pos, current_pos, children_pos, i,
                 "nonterminal"
             )
 
-        return [obl_case]
+        # The opposition leader has gone into hiding.
+        obl_mark = (
```

Comment:
we could refactor to generalize over all these handle_* functions, right? no need to do it right now, but maybe add an issue

Reply:
Yes, I agree. Submitted a new issue.